man/rgbgfx.1

```diff
@@ -103,6 +103,19 @@ and has the same size.
 Same as
 .Fl a Ar base_path Ns .attrmap
 .Pq see Sx Automatic output paths .
+.It Fl B Ar color , Fl \-background-color Ar color
+Set a background color to be omitted from output.
+Colors are accepted as
```

Comment:
```suggestion
Colors are accepted in
```

Using different wording should help highlight that `transparent` is meant literally.